smart_card_connector_app/src/background.js

```diff
@@ -109,6 +113,26 @@ function externalConnectionListener(port) {
 }
 
 /**
+ * @param {*} message
+ * @param {MessageSender} sender
```

Comment:
I believe this parameter is always non-null. E.g. see: https://cs.chromium.org/chromium/src/third_party/closure_compiler/externs/chrome_extensions.js?l=2332